openmrs/omod/src/main/java/org/openmrs/projectbuendia/webservices/rest/PatientResource.java

```diff
@@ -463,10 +460,6 @@ protected void applyEdits(Patient patient, SimpleObject edits) {
                     patient.setBirthdate(Utils.parseLocalDate((String) entry.getValue(), BIRTHDATE));
                     changedPatient = true;
                     break;
-                case GENDER:
-                    patient.setGender(normalizeSex((String) entry.getValue()));
-                    changedPatient = true;
-                    break;
```

Comment:
Not once the corresponding client change is checked in :) I'm taking the opportunity whilst doing the sync refactor to clean up TODOs that would result in breaking the protocol, because the sync refactor breaks the protocol anyway.